lucid-book-store/src/utility/alerts/alerts.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useRef, useEffect } from "react";
+import "./alert.css";
+
+const Alerts = ({ setAlertData, alertData }) => {
+  const { showAlert, alertMsg, alertType } = alertData;
+  const [timeOutId, setTimeoutId] = useState();
+
+  const toggleAlert = () => {
+    setAlertData({ ...alertData, showAlert: false });
+  };
+
+  useEffect(() => {
+    if (showAlert && !timeOutId) {
+      setTimeoutId(setTimeout(toggleAlert, 1500));
+    }
+  });
+
+  useEffect(() => {
+    timeOutId && clearTimeout(timeOutId);
+    setTimeoutId(setTimeout(toggleAlert, 1500));
+  }, [alertMsg]);
+
+  return (
+    <div
+      style={showAlert ? {} : { display: "none" }}
```

Comment:
you could remove the extra braces {} by using something like this instead:
!showAlert && {display: "none"}

Reply:
yup made the changes